signbank/dictionary/tasks.py

```diff
@@ -27,7 +29,7 @@ def move_glossvideo_to_valid_filepath(glossvideo):
     """
     old_file = glossvideo.videofile
     full_new_path = glossvideo.videofile.storage.get_valid_name(
-        glossvideo.videofile.name.split("/")[-1]
+        glossvideo.videofile.name.split("/")[-1].split("glossvideo.")[-1]
```

Comment:
**suggestion**: without looking at the storage class, it's unclear why this is splitting on glossvideo.; consider commenting?